src/test/java/com/foundation/salesforce/steps/OpportunitySteps.java

```diff
@@ -44,22 +49,79 @@
     public OpportunitySteps(final Context context) {
         this.context = context;
         opportunity = context.getOpportunity();
+        opportunityApi = OpportunityApi.getInstance();
     }
 
-    @Given("User log in into the Opportunities page")
-    public void LogInIntoTheOpportunitiesPage() {
-        opportunityApi.getInstance().getOpportunity();
-    }
-
-
+    /**
+     * Sets a map according to input map.
+     *
+     * @param inputBody that is input data
```

Comment:
Place a dot at the end of this line, and generally in all of the remaining parameter descriptions.